runs/comparisons/rf.py

```diff
@@ -39,13 +44,14 @@
         X_train=X_train,
         y_train=y_train,
         random_state=random_state,
-        n_calls=20,
+        n_calls=1000,
```

Comment:
Why are we using skopt and optuna based tuners?

Reply:
I don't know what the reason behind using both tuners was. If we only need one, I can remove the other (I'm guessing optuna is the one we prefer)

Reply:
as far as I see below we only call optuna, so we should definitely remove the other

Reply:
Removed